Library/ViewModels/SettingsNewslettersViewModelTests.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import XCTest
+import ReactiveSwift
+import Result
+import Prelude
+@testable import KsApi
+@testable import Library
+@testable import ReactiveExtensions_TestHelpers
+
+internal final class SettingsNewslettersViewModelTests: TestCase {
```

Comment:
Should there be a test here for the `didUpdateUser(user: User)` input firing the `self.currentUser` output?

Reply:
Good catch. Forgot that!